### PR TITLE
Add plugin hexo-graph

### PR DESCRIPTION
**Pull request by codepzj**
## Check List

**Please read and check followings before submitting a PR.**

- [ ] I want to publish my theme on Hexo official website.
  - [ ] I have read the [theme publishing doc](https://hexo.io/docs/themes#Publishing).
  - [ ] `name` is unique.
  - [ ] `link` URL is correct.
  - [ ] `preview` URL is correct.
  - [ ] `preview` URL web site is rendered correctly.
  - [ ] Add a screenshot to `source/themes/screenshots`.
  - [ ] Screenshot filename is same as value of `name`.
  - [ ] Screenshot size is `800 * 500`.
  - [ ] Screenshot file format is `png`.
- [x] I want to publish my plugin on Hexo official website.
  - [x] I have read the [plugin publishing doc](https://hexo.io/docs/plugins#Publishing).
  - [x] `name` is unique.
  - [x] `link` URL is correct.
- [ ] Others (Update, fix, translation, etc...)
  - Languages:
  - [ ] `en` English

Add plugin hexo-graph,A versatile plugin based on ECharts that integrates blog heatmap, monthly statistics chart, category statistics chart, and tag statistics chart.

Here is the repo:
- https://github.com/codepzj/hexo-graph

### CODE REVIEW

File: source/_data/plugins/hexo-graph.yml [hunk 1]
```diff
@@ -0,0 +1,9 @@
+description: A versatile plugin based on ECharts that integrates blog heatmap, monthly statistics chart, category statistics chart, and tag statistics chart.
+link: https://github.com/codepzj/hexo-graph
+tags:
+  - hexo
+  - echarts
+  - graph
+  - hexo-graph
+  - haohanxinghe
```
Comment (yoshinorin) [inline source/_data/plugins/hexo-graph.yml:8]: What is the `haohanxinghe`?

**Review by yoshinorin — COMMENTED** [review]
Thank you for the PR. Regarding the URL in the example, it is not accessible. Is it a valid URL?

File: source/_data/plugins/hexo-graph.yml [hunk 2]
```diff
@@ -0,0 +1,9 @@
+description: A versatile plugin based on ECharts that integrates blog heatmap, monthly statistics chart, category statistics chart, and tag statistics chart.
+link: https://github.com/codepzj/hexo-graph
+tags:
+  - hexo
+  - echarts
+  - graph
+  - hexo-graph
```
[the remark this answers is not in the document]
Comment (codepzj) [reply source/_data/plugins/hexo-graph.yml]: Okay, I have modified the relevant tags and submitted a new commit, and now I can see the image effect in my GitHub repository.😃

**Review by yoshinorin — APPROVED** [review]
LGTM 👍 Thank you :)

**Pull request merged**